src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const logo = document.querySelector('.logo');
+const message = document.createElement('div');
+
+message.classList.add('message');
+message.innerText = 'Promise was resolved!';
+
+const messageError = document.createElement('div');
+
+messageError.classList.add('message');
+messageError.classList.add('error-message');
+messageError.innerText = 'Promise was rejected!';
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve();
```

Comment:
The `resolve()` function is called without any arguments. If the intention is to append the message to the body upon resolution, consider passing the message element to `resolve()` and handling it in the `.then()` block.